src/main/java/org/snomed/snowstorm/core/data/domain/Relationship.java

```diff
@@ -62,11 +62,16 @@ public String getConceptId() {
 	@Size(min = 5, max = 18)
 	private String destinationId;
 
-	@JsonInclude(JsonInclude.Include.NON_EMPTY)
-	@JsonView(value = View.Component.class)
 	@Field(type = FieldType.Keyword)
+	@Size(min = 2, max = 4096)
+	@JsonIgnore
```

Comment:
You shouldn't need this. The absence of `@JsonInclude(JsonInclude.Include.NON_EMPTY)` should prevent it from being included in API responses and we want it included when writing to Elasticsearch.

Reply:
Fixed.